ConsoleApp/Stack.cs

```diff
@@ -0,0 +1,67 @@
+namespace ConsoleApp;
+
+class StackA<T>
+{
+    private record StackItem
+    {
+        public T Value { get; init; }
+        public StackItem Next { get; set; }
+        
+    }
+    private StackItem _head;
+    public int Length { get; private set; }
```

Comment:
it should be Count instead
```suggestion
    public int Count { get; private set; }
```